### PR TITLE
Cleanup of remove-and-blacklist cli command

### PR DESCRIPTION
**Pull request by Primdahl26**
remove useless try catch - add blacklist as optional argument - make function work when media is not in db

### CODE REVIEW

File: backend/cli.py [hunk 1]
```diff
@@ -151,40 +151,39 @@ async def providers_to_cache():
...
         )
 
+    if media:
         typer.echo(f"Removing and blacklisting: {media_id}")
         delete_media_by_id(db=db, media_id=media_id)
         typer.echo("Removed from database ✓")
 
-        with open("../blacklist.txt", "a+") as file:
+    if blacklist:
+        with open("..t/blacklist.txt", "a+") as file:
```
Comment (AndreasPB) [inline backend/cli.py:172]: Oopsies?

```suggestion
        with open("../blacklist.txt", "a+") as file:
```

Comment (Primdahl26) [reply backend/cli.py]: Ups my bad

**Comment by Primdahl26** [comment]
> I'll test this when I get home

image

File: backend/cli.py [hunk 2]
```diff
@@ -151,22 +151,24 @@ async def providers_to_cache():
...
 @app.command()
-def remove_and_blacklist(media_id: str):
-    try:
-        db = database.SessionLocal()
-        media = get_media_by_id(db=db, media_id=media_id)
-        if not media:
-            typer.echo(f"Cannot find media: {media_id}")
-            raise typer.Abort()
+def remove_media(media_id: str, blacklist: bool = True):
+    db = database.SessionLocal()
```
Comment (AndreasPB) [inline backend/cli.py:155]: We should probably add a docstring with what it does - `remove-media` doesn't really say much

**Review by AndreasPB — APPROVED** [review]

**Pull request merged**